src/main/java/seedu/duke/logic/commands/buscommand/RouteCommand.java

```diff
@@ -52,13 +72,9 @@ private void checkLocations(String[] locations) throws CustomException {
 
     private ArrayList<String> similarLocations(String[] locations) {
         ArrayList<String> possibleLocs = new ArrayList<>();
-        ArrayList<String> routeNames = new ArrayList<>();
-        for (BusStops info: EnumSet.allOf(BusStops.class)) {
-            routeNames.add(info.getName().toLowerCase());
-        }
-        if (!routeNames.contains(locations[0].trim().toLowerCase())) {
+        if (!isValidBusStop(locations[0].trim())) {
             possibleLocs = SimilarityCheck.similarLoc(locations[0]);
-        } else if (!routeNames.contains(locations[1].trim().toLowerCase())) {
+        } else if (!isValidBusStop(locations[1].trim())) {
             possibleLocs = SimilarityCheck.similarLoc(locations[1]);
         }
```

Comment:
Why is toLowerCase removed?

Reply:
isValidBusStop accounts for it. So this is redundant

Reply:
ahh ok